### PR TITLE
scan dir instead of expecting JSON file of filenames in transcribe.py

### PR DESCRIPTION
**Pull request by JRMeyer**

### CODE REVIEW

**Comment by kdavis-mozilla** [comment]
I'd guess that it's simply the documentation for the src parameter that is incorrect, not the code.

**Comment by reuben** [comment]
Also you can't just remove support for the catalog input, it's there for a reason, it's the output of the DSAlign tool.

**Comment by JRMeyer** [comment]
@tilmankamp -- I've put the `.catalog` functionality back into my branch, and rebased new commits from master.

can you check the logic of the `*.catalog` function? I didn't test for that, but I bet you have a catalog file somewhere you can test on

**Comment by rhamnett** [comment]
Please can we get this added back in? It's really handy. Thanks

**Comment by rhamnett** [comment]
@JRMeyer I have fixed transcribe.py in #2780 would you care to merge so we can get both scan dir and catalog usable? Thank you.

File: transcribe.py [hunk 1]
```diff
@@ -101,45 +108,58 @@ def resolve(base_path, spec_path):
...
+            # Transcribe all files in dir
+            if FLAGS.dst:
+                fail('Destination file not supported for batch decoding jobs.')
             else:
-                fail('Destination file "{}" already existing - use --force for overwriting'.format(dst_path), code=0)
-        elif os.path.isdir(os.path.dirname(dst_path)):
-            transcribe_one(src_path, dst_path)
-        else:
-            fail('Missing destination directory')
+                wav_paths = glob.glob(src_path + "/*.wav")
```
Comment (tilmankamp) [inline transcribe.py]: How about a `--recursive` option and then offering to glob `/**/*.wav`?

**Comment by rhamnett** [comment]
@JRMeyer polite reminder to rebase

**Comment by JRMeyer** [comment]
this got messy, making new PR

**Pull request closed without merging**